dashboard/migrations/0010_auto_20190809_1625.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-08-09 20:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0009_auto_20190731_1518'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='course',
+            old_name='term_id',
+            new_name='term',
+        ),
+        migrations.AddField(
+            model_name='course',
+            name='date_end',
+            field=models.DateTimeField(blank=True, null=True, verbose_name='End Date and Time'),
+        ),
+        migrations.AddField(
+            model_name='course',
+            name='date_start',
+            field=models.DateTimeField(blank=True, null=True, verbose_name='Start Date and Time'),
+        ),
+        migrations.AlterField(
+            model_name='academicterms',
+            name='date_end',
+            field=models.DateTimeField(blank=True, null=True, verbose_name='Term End Date and Time'),
+        ),
+        migrations.AlterField(
+            model_name='academicterms',
+            name='date_start',
+            field=models.DateTimeField(blank=True, null=True, verbose_name='Term Start Date and Time'),
+        ),
+        migrations.AlterField(
+            model_name='course',
+            name='id',
+            field=models.BigIntegerField(db_column='id', primary_key=True, serialize=False, verbose_name='Unizin Course Id'),
```

Comment:
Maybe drop the notion of "Unizin" here since MyLA might be used by non-Unizin schools?

Reply:
Done! See latest commit.